src/charm.py

```diff
@@ -674,7 +674,8 @@ def _start_services(self) -> None:
         )
         self.repo_check_systemd_service.write_text(service_content, encoding="utf-8")
 
-        execute_command(["/usr/bin/systemctl", "start", "repo-policy-compliance"])
+        execute_command(["/usr/bin/systemctl", "daemon-reload"])
+        execute_command(["/usr/bin/systemctl", "restart", "repo-policy-compliance"])
```

Comment:
I think this means that any existing tokens for spawned runners will be invalid, so we need to make sure to remove existing runners as well (perhaps this is already being done as a part of changing the token)